src/screens/OtherProfile/index.js

```diff
@@ -289,8 +290,8 @@ const OtherProfile = () => {
         <React.Fragment>
           {__renderFollowingButton()}
           <GlobalButton onPress={onCreateChat}>
-            <View style={styles.btnMsg}>
-              <EnveloveBlueIcon width={20} height={20} fill={colors.bondi_blue} />
+            <View style={styles.btnMsg(dynamicColor)}>
+              <IconEnvelope color={dynamicColor.primary} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `btnMsg` style now accepts `dynamicColor` as an argument, and the `EnveloveBlueIcon` has been replaced with `IconEnvelope`. Ensure that `dynamicColor` is correctly defined and passed where this style is used, and that `IconEnvelope` is correctly implemented.
```diff
- <View style={styles.btnMsg}>
-   <EnveloveBlueIcon width={20} height={20} fill={colors.bondi_blue} />
+ <View style={styles.btnMsg(dynamicColor)}>
+   <IconEnvelope color={dynamicColor.primary} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->